projects/plugins/jetpack/extensions/blocks/blogging-prompts/blogging-prompts.php

```diff
@@ -43,11 +43,14 @@ function inject_blogging_prompts() {
 
 	// And only for blogging sites or those explicitly responding to the prompt.
 	if ( should_load_blogging_prompts() ) {
-		$daily_prompts = wp_json_encode( jetpack_get_daily_blogging_prompts() );
+		$daily_prompts = jetpack_get_daily_blogging_prompts();
 
-		// See p7H4VZ-2cf-p2 for why prompt data is escaped this way.
 		if ( $daily_prompts ) {
-			wp_add_inline_script( 'jetpack-blocks-editor', 'var Jetpack_BloggingPrompts = JSON.parse( decodeURIComponent( "' . rawurlencode( $daily_prompts ) . '" ) );', 'before' );
+			wp_add_inline_script(
+				'jetpack-blocks-editor',
+				'var Jetpack_BloggingPrompts = ' . wp_json_encode( $daily_prompts, JSON_HEX_TAG ) . ';',
```

Comment:
If you want to be extra safe
```suggestion
				'var Jetpack_BloggingPrompts = ' . wp_json_encode( $daily_prompts, JSON_HEX_TAG | JSON_HEX_AMP ) . ';',
```

Reply:
Nice!